stratum/hal/lib/tdi/BUILD

```diff
@@ -278,7 +299,6 @@ stratum_cc_library(
 stratum_cc_library(
     name = "macros",
     hdrs = ["macros.h"],
-    defines = target_defines,
```

Comment:
Accidental deletion; fixed in a subsequent change.
macros.h #include bf_types.h.